quaddtype/numpy_quaddtype/_quaddtype_main.pyi

```diff
@@ -1,5 +1,5 @@
 from typing import Any, Literal, TypeAlias, final, overload
-
+import builtins
```

Comment:
Why the explicit import? We use `bool` without the `builtins.bool` elsewhere in the stubs